lms/djangoapps/instructor/views/api.py

```diff
@@ -1125,6 +1145,8 @@ def list_course_problems(request, course_id):
 @ensure_csrf_cookie
 @cache_control(no_cache=True, no_store=True, must_revalidate=True)
 @require_level('staff')
+@require_post_params(group_id="id of group to rename")
+@require_post_params(group_name="the new name of the group query with id group_id")
```

Comment:
pretty sure you can just combine these into one:
`@require_post_params(group_id="id of group to rename", group_name="the new name of the group query with id group_id")`